twitter/table_twitter_user.go

```diff
@@ -49,19 +50,34 @@ func listUser(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (
 		plugin.Logger(ctx).Error("twitter_user.listUser", "query_error", lookupErr, "opts", opts)
 		return nil, lookupErr
 	}
+
+	softError := []*twitter.ErrorObj{}
+
 	// Soft error, e.g. 404
 	if len(result.Raw.Errors) > 0 {
-		errMsgs := []string{}
 		for _, e := range result.Raw.Errors {
 			plugin.Logger(ctx).Error("twitter_user.listUser", "query_error", e, "opts", opts)
-			errMsgs = append(errMsgs, fmt.Sprintf("%s: %s", e.Title, e.Detail))
+			
+			// Check if the Not Found Error is a result of "pinned_tweet_id" parameter, return nil
 			if e.Title == "Not Found Error" {
+				if e.Parameter == "pinned_tweet_id" {
+					continue
+				}
 				return nil, nil
 			}
+			softError = append(softError, e)
+		}
+	}
+
+	if len(softError) > 0 {
```

Comment:
What is a "soft error" in the context of this table/API? Also, why do we gather all "soft errors" and then iterate through them again here? Is it just to avoid handling errors with `e.Parameter == "pinned_tweet_id"`?